spark/src/main/kotlin/com/adevinta/spark/components/toggles/Switch.kt

```diff
@@ -178,6 +173,14 @@ public fun SwitchLabelled(
         content = content,
     )
 }
+/**
+ * @property checkedIcon icon to be used for the thumb in checked state
+ * @property uncheckedIcon icon to be used for the thumb in unchecked state
+ */
+public data class IconState(
+    val checkedIcon: SparkIcon = SparkIcon.Toggles.Check.Simple,
```

Comment:
since we already have icon as the class name imo we could use only `checked` and `unchecked`